fake-files/output-expected/spraggins/spraggins.ims.json

```diff
@@ -1,8 +1,8 @@
 {
   "dimensions": [
     "mz",
-    "x",
-    "y"
+    "y",
+    "x"
```

Comment:
Originally I had the arrays stored as `[mz,x,y]` because the arrays printed in a manner that was easy to read. I hadn't considered how the ordering of the arrays themselves would benefit our needs in Vitessce specifically (favoring x as the row-major axis). 

read: The mz channel was always first but was due to a happy accident; this PR formalizes how arrays should be ordered in the future (optimized 2D access).